s2spy/time.py

```diff
@@ -314,13 +317,11 @@ def __init__(self, anchor: Tuple[int, int] = (11, 30)):
         """
         self.month = anchor[0]
         self.day = anchor[1]
-        self._list_target = []
-        self._list_precursor = []
+        self._target = []
+        self._precursor = []
         self._total_length_target = 0
```

Comment:
I think the plural `_targets` and `_precursors` would be better.

Reply:
Good spotting! Corrected!